src/Commands/RunRepeater.ts

```diff
@@ -167,24 +167,28 @@ export class RunRepeater implements CommandModule {
               timeout: args.timeout as number,
               proxyUrl: (args.proxyInternal ?? args.proxy) as string,
               certs: args.cert as Cert[],
+              maxBodySize: 100 * 1024,
```

Comment:
imo better initialize it with `Infinity` to not introduce new debugging point when behavior of new cli running with old/buggy server version may behave somewhat different